G1ANT.Addon.Net/G1ANT.Addon.Net/NetAddon.cs

```diff
@@ -27,5 +27,38 @@ namespace G1ANT.Addon.Net
     [CommandGroup(Name = "vnc",  Tooltip = "A command connected with a VNC server.")]
     public class NetAddon : Language.Addon
     {
+        public override void Initialize()
+        {
+            base.Initialize();
+        }
+
+        public override void LoadDlls()
+        {
+            UnpackDrivers();
+            base.LoadDlls();
+        }
+
+        private void UnpackDrivers()
+        {
+            var unpackfolder = AbstractSettingsContainer.Instance.UserDocsAddonFolder.FullName;
+            var driversDictionary = new Dictionary<string, byte[]>()
+            {
+                { "putty.exe", Resources.putty },
+                { "VNC.exe", Resources.VNC },
+                { "telnet.exe", Resources.telnet },
+            };
+            foreach (var exe in driversDictionary.Where(e => !File.Exists(Path.Combine(unpackfolder, e.Key))
+            || e.Value.Length !=new FileInfo(Path.Combine(unpackfolder, e.Key)).OpenRead().Length))
```

Comment:
CTRL+K CTRL+D

Reply:
????

Reply:
done

Reply:
`FileInfo` should already contain size of the file, you don't need to open it.
DRY - `Path.Combine(unpackfolder, e.Key)`; move the lambda (or some parts of it) into a method.
`driversDictionary` with `exe` elements => `driversDictionary` => `drivers`, `exe` => `driver`

Reply:
done